content/includes/business-logos.html

```diff
@@ -0,0 +1,19 @@
+<section id="business-logo">
+    <div class="ui very padded container">
+        <div class="ui four column grid">
```

Comment:
This should be `.ui.stackable.grid`, or it will try to display inline on mobile viewports:

![image](https://github.com/readthedocs/website/assets/1140183/0927f96c-2c46-40b5-b464-52b53bd86783)